it/controllers/IndexControllerISpec.scala

```diff
@@ -466,14 +796,80 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
               status = SubmissionStatusEnum.Submitted
             )
           )
-        )),
+          )),
         communications = Seq.empty,
         financial = None
       )
     ),
     otherPenalties = Some(true)
   )
 
+  val getPenaltyDetailsPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue: GetPenaltyDetails = getPenaltyDetailsPayloadWithAddedPoint.copy(
+    totalisations = Some(Totalisations(
+      LSPTotalValue = Some(400),
+      penalisedPrincipalTotal = Some(121.40),
+      LPPPostedTotal = Some(165.25),
+      LPPEstimatedTotal = Some(46.55),
+      LPIPostedTotal = Some(40.55),
+      LPIEstimatedTotal = Some(6)
+    )),
+
+    lateSubmissionPenalty = Some(LateSubmissionPenalty(
+      summary = LSPSummary(
+        activePenaltyPoints = 2,
+        regimeThreshold = 4,
+        inactivePenaltyPoints = 0,
+        penaltyChargeAmount = 0
+      ),
+      details = Seq(
+        LSPDetails(
+          penaltyNumber = "1234567890",
+          penaltyOrder = "01",
+          penaltyCategory = LSPPenaltyCategoryEnum.Point,
+          penaltyStatus = LSPPenaltyStatusEnum.Active,
+          FAPIndicator = Some("X"),
+          penaltyCreationDate = sampleDate1V2,
+          penaltyExpiryDate = sampleDate1V2.plusMonths(1).plusYears(2),
+          expiryReason = None,
+          communicationsDate = sampleDate1V2,
+          lateSubmissions = Some(Seq(LateSubmission(
+            taxPeriodStartDate = Some(sampleDate1V2),
+            taxPeriodEndDate = Some(sampleDate1V2),
+            taxPeriodDueDate = Some(sampleDate1V2),
+            returnReceiptDate = Some(sampleDate1V2),
+            taxReturnStatus = TaxReturnStatusEnum.Fulfilled))),
+          appealInformation = Some(Seq(AppealInformationType(Some(models.v3.appealInfo.AppealStatusEnum.Unappealable), None))),
+          chargeAmount = None,
+          chargeOutstandingAmount = Some(BigDecimal(100)),
+          chargeDueDate = None
+        ),
+        LSPDetails(
+          penaltyNumber = "1234567890",
+          penaltyOrder = "01",
+          penaltyCategory = LSPPenaltyCategoryEnum.Point,
+          penaltyStatus = LSPPenaltyStatusEnum.Active,
+          FAPIndicator = Some("X"),
```

Comment:
Wouldn't this mean it was an added point? (if not intended, set to None)

Reply:
Done, set to None